src/server.c

```diff
@@ -294,30 +294,65 @@ size_t dictSdsKeyLen(dict *d, const void *key) {
     return sdslen((sds)key);
 }
 
-int dictSdsKeyCompareWithLen(dict *d, const void *key1, const size_t l1,
-                                      const void *key2, const size_t l2)
+static uint64_t dictHashKV(const void *kv) {
+    sds sdsKey = kvobjGetKey((kvobj *) kv);
+    return dictGenHashFunction(sdsKey, sdslen(sdsKey));
+}
+
+int dictCompareKV(dictCmpCache *c, const void *kv1, const void *kv2)
 {
```

Comment:
```suggestion
int dictCompareKV(dictCmpCache *c, const void *kv1, const void *kv2) {
```

btw, what about naming c as cmp_cache? c is easily regarded as a client at first glance.

Reply:
renamed to `cache`